app/Plugin/CoreApi/Config/routes.php

```diff
@@ -29,6 +29,7 @@
 // Note REST API v2 has taken the form /api/v2/objects
 
 // COmanage CO Person Read API
+// /api/co/:coid/core/v1/people#show
```

Comment:
Did you mean to commit these comments? (I'm not sure what the anchor tags like `#show` are referring to.)

Reply:
No. I will remove them.